utils_cv/detection/data.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from typing import List
+from urllib.parse import urljoin
+
+
+class Urls:
+    # for now hardcoding base url into Urls class
+    base = (
+        "https://cvbp.blob.core.windows.net/public/datasets/object_detection/"
+    )
+
+    # traditional datasets
+    fridge_objects_path = urljoin(base, "odFridgeObjects.zip")
+
+
+def get_coco_labels() -> List[str]:
+    """ List of Coco labels with the original idexing.
+
+    Reference: https://github.com/pytorch/vision/blob/master/docs/source/models.rst
+
+    Returns:
+        Coco labels
+    """
+
+    return [
+        "__background__",
+        "person",
+        "bicycle",
+        "car",
+        "motorcycle",
+        "airplane",
+        "bus",
+        "train",
+        "truck",
+        "boat",
+        "traffic light",
+        "fire hydrant",
+        "N/A",
+        "stop sign",
+        "parking meter",
+        "bench",
+        "bird",
+        "cat",
+        "dog",
+        "horse",
+        "sheep",
+        "cow",
+        "elephant",
+        "bear",
+        "zebra",
+        "giraffe",
+        "N/A",
+        "backpack",
+        "umbrella",
+        "N/A",
```

Comment:
Are the N/A's expected?

Reply:
not sure.. but i copied from the official pytorch docs: https://github.com/pytorch/vision/blob/master/docs/source/models.rst#object-detection-instance-segmentation-and-person-keypoint-detection